tests/test-land-patterns.stanza

```diff
@@ -12,6 +12,37 @@ defpackage ocdb/test/land-patterns:
   import ocdb/rules
   import ocdb/generic-components
   import ocdb/symbols
+  import ocdb/land-protrusions
+
+pcb-component QFP:
+  pcb-landpattern LQFP48:
+    make-qfp-landpattern(48,   ; number of pins 
+                         0.5,  ; pitch ("e")
+                         tol(9.0, 0.2),   ; outer size (lead-to-lead), "D" and "E"
+                         tol(7.0, 0.2),   ; package size, "D1" and "E1"
+                         tol(0.6, 0.15),  ; lead size, x direction (for p[1]), "L"
+                         tol(0.22, 0.05)) ; lead size, y direction (for p[1]), "b"
+  port p: pin[1 through 48]
+  landpattern = LQFP48(
+    for n in 1 through 48 do:
+      p[n] => LQFP48.p[n]
+  )
+  make-box-symbol()
+
+pcb-component QFN:
+  pcb-landpattern MLPQ-UT8:
+    make-qfn-landpattern([3, 1, 3, 1],    ; number of pins on each side (Up, Down, Left, Right)
+                         0.4,             ; pitch
+                         tol(1.5, 0.0),   ; package dimensions 
+                         tol(0.35, 0.05), ; lead size, x direction (for p[1])
+                         tol(0.20, 0.05)) ; lead size, y direction (for p[1])
+
+  port p: pin[1 through 8]
+  landpattern = MLPQ-UT8(
+    for n in 1 through 8 do:
+      p[n] => MLPQ-UT8.p[n]
```

Comment:
I am curious about if `map({p[_0] => MLPQ-UT8.p[_0]}, 1 through 8)` is valid

Reply:
I think `map` is lazy so that wouldn't work necessarily

Reply:
Bertrand is a quick learner with `_0`!
Here you can use `do` instead of `map`. The key-value pairs aren't actually returned, they are macros to ESIR calls.

Reply:
I would keep it as is -- the way Mike wrote it follows our ESIR design standards since curlies aren't beginner-friendly.